### DISCUSSION TITLE
Allow fifths (or even custom scales)

### DISCUSSION THREAD

**Discussion by knaftN** [rxhanson/Rectangle#1023] [Ideas]
Hi there and thanks for the awesome work!

As a widescreen user, I like to have two apps open next to each other, but sometimes the `1/2 + 1/2` or `1/3 + 2/3` are always a bit "off".
Proposal: Allow the user setting something like `2/5 + 3/5`. I would really dig this feature 🥳

**Comment by Aaron-Rumpler** [comment]
[Rectangle Pro](https://rectangleapp.com/pro) supports defining custom sizes and snapping areas. As for fifths in Rectangle, if someone wants to add it they can submit a PR (this was done for eighths (#761) and ninths (#702) a while back), but it's not something that would make it into the GUI (@rxhanson tends to keep Rectangle more lightweight, so it would be a terminal command to bind shortcuts to it, as opposed to something that would show in the dropdown menu or be configurable from the GUI).
